src/cellmanager.js

```diff
@@ -527,7 +527,7 @@ export default class CellManager {
                     const done = editor.setValue(value, rowIndex, col);
 
                     // update cell immediately
-                    this.updateCell(colIndex, rowIndex, value);
+                    this.updateCell(colIndex, rowIndex, value, true);
```

Comment:
added this for the case when someone edits a cell, as during editing we directly update the cell's content property - this will make sure to refresh the cell's html as well